Source/WebKit/WebKit.xcodeproj/project.pbxproj

```diff
@@ -17494,17 +17497,6 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 		};
-		650994CE28C942B7006EE6DB /* Sources */ = {
-			isa = PBXSourcesBuildPhase;
-			buildActionMask = 2147483647;
-			files = (
-				650994CF28C942B7006EE6DB /* AuxiliaryProcessMain.cpp in Sources */,
-				413E5C9429B15174002F4987 /* CacheStorageCache.cpp in Sources */,
-				E31586D12AD610F30062ED2A /* ExtensionEventHandler.mm in Sources */,
-				650994D028C942B7006EE6DB /* XPCServiceMain.mm in Sources */,
-			);
-			runOnlyForDeploymentPostprocessing = 0;
-		};
```

Comment:
This is removing the sources build phase for the WebContent.Crashy target, which doesn't seem at all related.

Reply:
Good catch, will fix!